packages/react/src/isomorphicClerk.ts

```diff
@@ -85,95 +77,17 @@ type MethodName<T> = {
 
 type MethodCallback = () => Promise<unknown> | unknown;
 
+type WithVoidReturn<F extends (...args: any) => any> = (
+  ...args: Parameters<F>
+) => ReturnType<F> extends Promise<infer T> ? Promise<T | void> : ReturnType<F> | void;
+type WithVoidReturnFunctions<T> = {
+  [K in keyof T]: T[K] extends (...args: any) => any ? WithVoidReturn<T[K]> : T[K];
+};
```

Comment:
Most methods were adjust to add `| void` to the return type, so I've gone ahead and created a helper type to do that automatically.